backend/src/models/articleManager.js

```diff
@@ -0,0 +1,9 @@
+const AbstractManager = require("./AbstractManager");
```

Comment:
this file exports a class: it should be named after the class (in UpperCamelCase)